api/Controllers/spaceController.js

```diff
@@ -0,0 +1,23 @@
+import { sqlPool } from '../Utils/DbUtils.js';
+
+export const createSpace = async (req, res) => {
+    var query = 'call insert_space($1,$2)';
+    var params = [req.body.Space, req.params.orgName];
```

Comment:
Maybe get everything you need added to the body since it's a post rather than using both query params and the body